h2o-dist/index.html

```diff
@@ -545,7 +545,9 @@ <h2>Use H<sub>2</sub>O directly from Python</h2>
                 <p class="terminal" id="to_copy8">
                     pip install requests<br/>
                     pip install tabulate<br/>
-                    pip install future
+                    pip install future<br/>
```

Comment:
Does the copy-to-clipboard feature still work?

Should it instead be 2 cells? Since the second one is optional?

Reply:
Yeah maybe two cells so that copy to clipboard will work on each one.